Source/SwiftLintFramework/Rules/ConditionalReturnsOnNewline.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  ConditionalReturnsOnNewline.swift
+//  SwiftLint
+//
+//  Created by Rohan Dhaimade on 12/08/2016.
+//  Copyright © 2016 Realm. All rights reserved.
+//
+
+import Foundation
+import SourceKittenFramework
+
+public struct ConditionalReturnsOnNewline: Rule, OptInRule {
```

Comment:
Could you please make this conform to `ConfigurationProviderRule` with a `public var configuration = SeverityConfiguration(.Warning)` property? See [ForceUnwrappingRule.swift](https://github.com/realm/SwiftLint/blob/master/Source/SwiftLintFramework/Rules/ForceUnwrappingRule.swift) as an example.

You also don't need the `init(configuration:)` below.